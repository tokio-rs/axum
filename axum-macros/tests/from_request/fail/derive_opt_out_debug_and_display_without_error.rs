use axum_macros::FromRequest;

#[derive(FromRequest)]
#[from_request(rejection_derive(!Debug, !Display))]
struct Extractor {
    body: String,
}

fn main() {}
