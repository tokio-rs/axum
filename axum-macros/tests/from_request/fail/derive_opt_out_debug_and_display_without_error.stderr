error: opt out of `Debug` and `Display` requires also opting out of `Error`. Use `#[from_request(rejection_derive(!Debug, !Display, !Error))]`
 --> tests/from_request/fail/derive_opt_out_debug_and_display_without_error.rs:4:34
  |
4 | #[from_request(rejection_derive(!Debug, !Display))]
  |                                  ^^^^^
